app/models/manageiq/providers/kubernetes/container_manager/scanning/job.rb

```diff
@@ -54,18 +54,20 @@ def start
       :pod_namespace   => namespace
     ))
 
-    pod = pod_definition
-    _log.info("creating pod #{pod_full_name} to analyze docker image " \
-              "#{options[:docker_image_id]}: #{pod.to_json}")
-
+    log_pref = nil
     begin
+      log_pref = "Getting Service Account #{IMAGE_INSPECTOR_SA} for pod [#{pod_full_name}]"
```

Comment:
@moolitayer This is a little misleading because the next command will get the pod definition which contains the service account. I am not sure that this is very necessary..